config-model/src/main/java/com/yahoo/vespa/model/content/ContentSearchCluster.java

```diff
@@ -310,7 +310,7 @@ public void handleRedundancy(Redundancy redundancy) {
     @Override
     public void getConfig(ProtonConfig.Builder builder) {
         double visibilityDelay = hasIndexedCluster() ? getIndexed().getVisibilityDelay() : 0.0;
-        builder.feeding.concurrency(0.30); // As if specified 0.6 in services.xml
+        builder.feeding.concurrency(0.35); // As if specified 0.7 in services.xml
```

Comment:
Is this dependent of a default in proton.def? If so, consider doing the same as here: https://github.com/vespa-engine/vespa/blob/35283b454dfdaa9acbcea24274e01e65d128b7b6/config-model/src/main/java/com/yahoo/vespa/model/search/NodeFlavorTuning.java#L111

Reply:
No, it is not. This is itself a scaling factor, and does not rely on default in def file.
I just did it as it was done earlier on.